tests/unit/rpc/handlers/AccountNFTsTests.cpp

```diff
@@ -402,6 +403,42 @@ TEST_F(RPCAccountNFTsHandlerTest, Marker)
     });
 }
 
+TEST_F(RPCAccountNFTsHandlerTest, InvalidMarker)
+{
+    backend->setRange(MINSEQ, MAXSEQ);
+    auto const ledgerHeader = CreateLedgerHeader(LEDGERHASH, MAXSEQ);
+    EXPECT_CALL(*backend, fetchLedgerBySequence).Times(1);
+    ON_CALL(*backend, fetchLedgerBySequence).WillByDefault(Return(ledgerHeader));
+
+    auto const accountObject = CreateAccountRootObject(ACCOUNT, 0, 1, 10, 2, TXNID, 3);
+    auto const accountID = GetAccountIDWithString(ACCOUNT);
+    ON_CALL(*backend, doFetchLedgerObject(ripple::keylet::account(accountID).key, 30, _))
+        .WillByDefault(Return(accountObject.getSerializer().peekData()));
+
+    auto const pageObject =
```

Comment:
I think when marker is invalid, handler will return error. These mock will not be used.